entity-type-creator/components/EntityTypeFieldEditor.tsx

```diff
@@ -433,7 +431,7 @@ export default function EntityTypeFieldEditor({
               </Grid>
             </>
           ),
-          [field.valueGetter]
```

Comment:
These were causing for changes in the code editor to reset the field on edit, as it kept an old copy of `field`.